tests/c#/10/BinaryLoaderTests.cs

```diff
@@ -182,7 +181,6 @@ public void HelperWorking()
 		Assert.Equal(new byte[] { (byte)'b', 0b01000100 }, rawCode2);
 	}
 
-
```

Comment:
Ok so i admit that this is is very hard to know, but there is a reason why there are sometimes two empty lines. Vertical space separates unrelated sections. Here for example I separate the helper and the test for the helper from the actual tests with two lines, instead of just one like the separate test methods. 

Yes im aware that i could use `#region`s, comments or split files to do this, but this is just the way i do it. I appreciate you cleaning up the whitespaces, but while some of them are actually just just wrong it is not trivial to know which of them are. Also since this is such a minute improvement even if you get it right i would encourage you to not spend too much time on it, especially since you actually added some messed up spacing in other places.